app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+class InvalidCommandError(Exception):
+    ...
+
+
+def move_file(command: str) -> None:
+    command_name, origin_path, new_path = command.split()
```

Comment:
you must do it after 11 line because if len(command.split()) != 3 code will be broken